api-catalog-ui/frontend/src/components/Wizard/WizardComponents/WizardInputs.test.jsx

```diff
@@ -320,94 +334,103 @@ describe('>>> WizardInputs tests', () => {
+                    scheme: { value: 'schemeValue', question: 'Who?' },
+                },
+            ],
         };
-        const wrapper = enzyme.shallow(
-            <WizardInputs data={dummyData} />
-        );
+        const wrapper = enzyme.shallow(<WizardInputs data={dummyData} />);
         const instance = wrapper.instance();
-        const result = instance.renderInputs(dummyData.content,);
+        const result = instance.renderInputs(dummyData.content);
         expect(result[0]).not.toEqual(null);
     });
     it('should correctly generate captions for all restrictions', () => {
         const dummyData = {
             text: 'Category',
-            content: [{
-                test: {
-                    value: '',
-                    question: 'Why?',
-                    optional: true,
-                    maxLength: 40,
-                    lowercase: true,
-                    empty: true,
-                    problem: false
+            content: [
+                {
+                    test: {
+                        value: '',
+                        question: 'Why?',
+                        optional: true,
+                        maxLength: 40,
+                        lowercase: true,
+                        empty: true,
+                        problem: false,
+                    },
                 },
-            }],
+            ],
         };
-        const wrapper = enzyme.shallow(
-            <WizardInputs data={dummyData} />
-        );
+        const wrapper = enzyme.shallow(<WizardInputs data={dummyData} />);
         const instance = wrapper.instance();
         instance.renderInputElement('test', 0, dummyData.content[0].test);
-        expect(wrapper.find('FormField').props().caption).toEqual('Optional field; Field must be lowercase; Max length is 40 characters');
+        expect(wrapper.find('FormField').props().caption).toEqual(
+            'Optional field; Field must be lowercase; Max length is 40 characters'
+        );
     });
-    it('should handle select\'s onCLick', () => {
+    it("should handle select's onCLick", () => {
         const updateWizardData = jest.fn();
         const dummyCategory = {
             text: 'Category 1',
-            content: [{
-                type: { value: 'val', question: 'Why?', options: ['Option 1', 'val'] },
-            }],
+            content: [
+                {
+                    type: { value: 'val', question: 'Why?', options: ['Option 1', 'val'] },
+                },
+            ],
             interference: 'catalog',
         };
         const wrapper = enzyme.shallow(
-            <WizardInputs updateWizardData={updateWizardData}  validateInput={jest.fn()} tiles={['TIle 1']} data={dummyCategory}/>
+            <WizardInputs
+                updateWizardData={updateWizardData}
+                validateInput={jest.fn()}
+                tiles={['TIle 1']}
```

Comment:
Could we also fix this indenting? It's here and above.

Reply:
@balhar-jakub I don't see the indenting issue you're talking about? Do you want the `WizardsInput` component on one line?